index.js

```diff
@@ -9,24 +9,38 @@ module.exports = (opts) => {
     quality = 2,
     offset = 0,
     input,
-    output
+    output,
+    noaccurate = false
   } = opts
 
   return new Promise((resolve, reject) => {
     const cmd = ffmpeg(input)
-      .seek(offset / 1000)
+
+    const inputOptions = []
+    if (noaccurate) {
+      inputOptions.push('-ss', offset / 1000)
+    } else {
+      cmd.seek(offset / 1000)
+    }
+
+
+    if (noaccurate) {
+      inputOptions.push('-noaccurate_seek')
+    }
 
     const outputOptions = [
       '-vframes', 1,
       '-q:v', quality
     ]
 
-    cmd
-      .outputOptions(outputOptions)
-      .output(output)
-      .on('start', (cmd) => log && log({ cmd }))
-      .on('end', () => resolve())
-      .on('error', (err) => reject(err))
-      .run()
+    cmd.
```

Comment:
please revert this change

Reply:
I blame my IDE!  Sorry about that.  Cheers